RadixWallet/Core/FeaturePrelude/AddressView/AddressDetails+View.swift

```diff
@@ -94,22 +94,35 @@ public extension AddressDetails {
 			.animation(.easeInOut, value: store.qrImage.isSuccess)
 		}
 
+		@ViewBuilder
 		private var actions: some SwiftUI.View {
-			HStack(spacing: .large3) {
-				Button(L10n.AddressDetails.copy, image: .copy) {
+			FlowLayout(multilineAlignment: .center, spacing: .large3) {
+				actionButton(L10n.AddressDetails.copy, image: .copyMedium) {
```

Comment:
If we are already resizing the image, why not use `copy-big` (that is 24x24`) and avoid adding a new asset?